### PR TITLE
#62 about page fixes

### LINKED ISSUE #62: About us page fixes
## About us page
- Video padding top 64px
- OurValues title padding bottom 64px
- OurValues: use `flex-wrap` instead of `flex-col`
- MeetOurTeam pb 40px on people cards
- change pics/names for MeetOurTeam. Add links for this as well

### PR DESCRIPTION
**Pull request by JagrajAulakh**
Closes #62

### CODE REVIEW

File: src/components/AboutUsPage/MeetTheTeam.js [hunk 1]
```diff
@@ -9,39 +16,48 @@ const MeetTheTeam = () => {
...
-			<div className="flex flex-wrap justify-center">
+			<div className="max-w-screen-xl mx-auto flex flex-wrap justify-around">
 				{teamMembers.map((info, index) => (
-					<div
-						className="flex flex-col items-center px-10"
+					<a
+						href={
+							info.link
+								? info.link
+								: "https://www.youtube.com/watch?v=dQw4w9WgXcQ"
```
Comment (hassanuahmad) [inline src/components/AboutUsPage/MeetTheTeam.js:55]: might wanna remove this

File: src/components/AboutUsPage/MeetTheTeam.js [hunk 2]
```diff
@@ -9,39 +16,48 @@ const MeetTheTeam = () => {
...
+						}
+						target="_blank"
+						className={`flex flex-col items-center w-40 p-4 my-2 mx-3
```
Comment (hassanuahmad) [inline src/components/AboutUsPage/MeetTheTeam.js]: mx-3 might be a problem, could we either bump it up to mx-4 or 2

File: src/components/AboutUsPage/MeetTheTeam.js [hunk 3]
```diff
@@ -9,39 +16,48 @@ const MeetTheTeam = () => {
...
+							rounded-lg shadow-xl transition-color duration-200
+							hover:bg-gray-200`}
```
Comment (hassanuahmad) [inline src/components/AboutUsPage/MeetTheTeam.js:60]: would you wanna rename gray-200 so we can have as a set color

File: src/components/AboutUsPage/MeetTheTeam.js (hunk 2 above)
Comment (JagrajAulakh) [reply src/components/AboutUsPage/MeetTheTeam.js]: I will change to `mx-2`

File: src/components/AboutUsPage/MeetTheTeam.js (hunk 1 above)
Comment (JagrajAulakh) [reply src/components/AboutUsPage/MeetTheTeam.js:55]: no 😈

File: src/components/AboutUsPage/MeetTheTeam.js (hunk 2 above)
Comment (JagrajAulakh) [reply src/components/AboutUsPage/MeetTheTeam.js]: done

File: src/components/AboutUsPage/MeetTheTeam.js (hunk 3 above)
Comment (JagrajAulakh) [reply src/components/AboutUsPage/MeetTheTeam.js:60]: does this matter? can't we rely on gray-200?

Comment (hassanuahmad) [reply src/components/AboutUsPage/MeetTheTeam.js:60]: would be easier to say something like hover-color or something, just makes it easy to read

Comment (hassanuahmad) [reply src/components/AboutUsPage/MeetTheTeam.js:60]: but it kinda is extra work to do

**Review by hassanuahmad — APPROVED** [review]

**Pull request merged**